addons/mrp/tests/test_stock_report.py

```diff
@@ -467,3 +467,68 @@ def test_mo_overview_same_component(self):
         self.assertEqual(len(repl1), 1)
         self.assertEqual(repl0[0]['summary']['quantity'], 3)
         self.assertEqual(repl1[0]['summary']['quantity'], 5)
+
+    def test_mrp_report_overview(self):
+        product_parent = self.env['product.product'].create({
+            'name': 'Choco Cake',
+            'type': 'product',
+            'route_ids': [Command.set([self.ref('mrp.route_warehouse0_manufacture')])],
+        })
+        product_child = self.env['product.product'].create({
+            'name': 'Double Choco Cake',
+            'type': 'product',
+            'route_ids': [Command.set([self.ref('mrp.route_warehouse0_manufacture')])],
+        })
+
+        warehouse_1 = self.env.ref('stock.warehouse0')
+
+        # reordering rule
+        self.env['stock.warehouse.orderpoint'].create({
+            'name': 'Child RR',
+            'location_id': warehouse_1.lot_stock_id.id,
+            'product_id': product_child.id,
+            'product_min_qty': 0,
+            'product_max_qty': 0,
+        })
+        child_bom = self.env['mrp.bom'].create([{  # child
+            'product_id': product_child.id,
+            'product_tmpl_id': product_child.product_tmpl_id.id,
+            'product_qty': 1,
+        }])
+        parent_bom = self.env['mrp.bom'].create([{  # parent
+            'product_id': product_parent.id,
+            'product_tmpl_id': product_parent.product_tmpl_id.id,
+            'product_qty': 1,
+            'bom_line_ids': [
+                Command.create({'product_id': product_child.id, 'product_qty': 1}),
+            ],
+        }])
+        mo = self.env['mrp.production'].create({
+            'product_id': product_parent.id,
+            'product_qty': 6.0,
+            'bom_id': parent_bom.id,
+            'picking_type_id': warehouse_1.manu_type_id.id,
+        })
+        mo.action_confirm()
+
+        child_mo = self.env['mrp.production'].search(['&', ('product_id.id', '=', product_child.id), ('bom_id.id', '=', child_bom.id)], limit=1)
```

Comment:
Nitpiiiiick

```suggestion
        child_mo = self.env['mrp.production'].search([('product_id.id', '=', product_child.id), ('bom_id.id', '=', child_bom.id)], limit=1)
```

The default operator is _AND_